src/cryptography/hazmat/bindings/openssl/binding.py

```diff
@@ -23,11 +24,25 @@ def _osrandom_rand_status():
     return 1
 
 
+class _ConditionalLibrary(object):
```

Comment:
sooo, for reasons that I don't want to talk about, this will reduce performance on pypy.

Can you please write this as:

```py
import types

lib = types.ModuleType("name")
# add the attrs to lib here
```